### PR TITLE
Moved time slider, set slider default to max range

### PR DESCRIPTION
**Pull request by siennawishart**
Default slider bug: 
- there was an issue with useState initializing timeRange to 0 instead of maxMonths, i.e. initializing before maxMonths gets assigned its data so I added a useeffect to fix this
- defaulting the slider to 0 and maxMonths should only query reports so I adjusted index.ts

### CODE REVIEW

File: frontend/src/components/ListPanel.tsx [hunk 1]
```diff
@@ -74,6 +75,7 @@ function ListPanel({
 
   const handleTimeRangeChange = (event: Event, newValue: number | number[]) => {
     setTimeRange(newValue as number[]);
+    console.log('test newValue: ', newValue);
```
Comment (MickeyDang) [inline frontend/src/components/ListPanel.tsx:78]: .

**Review by MickeyDang — COMMENTED** [review]
Some quick questions.

**Comment by siennawishart** [comment]
> Some quick questions.
> 
> Can you also post a screenshot of it working? The vercel preview doesn't seem to be showing that, but idk if it's because it requires your backend changes to be deployed for it to work as well.

Here's a screenshot: 
image

File: frontend/src/components/ListPanel.tsx [hunk 2]
```diff
@@ -119,25 +132,25 @@ function ListPanel({
...
-        classes={{
-          markLabel: 'fieldnotes-mark-label',
-        }}
-      />
       <Row>
         <Accordion defaultActiveKey={['reports', 'relationships']} flush alwaysOpen>
           <ContextAwareToggle textBody="Reports" numItems={reportResults.length} eventKey="reports" />
           <Accordion.Collapse className="section-body" eventKey="reports">
             <Card.Body>
+              <Slider
```
[the remark this answers is not in the document]
Comment (siennawishart) [reply frontend/src/components/ListPanel.tsx:140]: Checked with her, she's good with it as seen in the screenshot

File: frontend/src/components/ListPanel.tsx (hunk 1 above)
Comment (siennawishart) [reply frontend/src/components/ListPanel.tsx:78]: will remove

**Review by MickeyDang — APPROVED** [review]
LGTM ✅

**Pull request merged**